internal/application/command.go

```diff
@@ -248,28 +250,34 @@ func writeDeviceResource(device models.Device, resourceName string, attributes s
 	err := driver.HandleWriteCommands(device.Name, device.Protocols, reqs, []*sdkModels.CommandValue{cv})
 	if err != nil {
 		errMsg := fmt.Sprintf("error writing DeviceResource %s for %s", dr.Name, device.Name)
-		return errors.NewCommonEdgeX(errors.KindServerError, errMsg, err)
+		return nil, errors.NewCommonEdgeX(errors.KindServerError, errMsg, err)
 	}
 
-	return nil
+	// from EdgeX3.0, updated resource value will be published to messageBus as long as
```

Comment:
Code is on 3.0 branch so no need to state this.

```suggestion
	// Updated resource value will be published to MessageBus as long as
```